lib/new_relic/agent/instrumentation/action_controller_other_subscriber.rb

```diff
@@ -0,0 +1,46 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require 'new_relic/agent/instrumentation/notifications_subscriber'
+require 'new_relic/agent/instrumentation/ignore_actions'
+require 'new_relic/agent/parameter_filtering'
+
+module NewRelic
+  module Agent
+    module Instrumentation
+      class ActionControllerOtherSubscriber < NotificationsSubscriber
+        def start_segment(name, id, payload)
+          segment = Tracer.start_segment(name: metric_name(name, payload))
+
+          add_segment_params(segment, payload, name)
+          push_segment(id, segment)
+        end
+
+        def add_segment_params(segment, payload, name)
+          segment.params[:filter] = payload[:filter] if payload[:filter]
+          segment.params[:keys] = payload[:keys] if payload[:keys]
+          segment.params[:original_path] = payload[:request].original_fullpath if payload[:request]
+
+          if payload[:context]
+            segment.params[:action] = payload[:context][:action]
+            segment.params[:controller] = payload[:context][:controller].to_s
+          end
+        end
+
+        def metric_name(name, payload)
+          controller_path = controller_name_for_metric(payload)
+          "Ruby/ActionController#{"/#{controller_path}" if controller_path}/#{name.gsub(/\.action_controller/, '')}"
```

Comment:
Conditionally including the extra `/` delimiter is very slick! One consideration here - there might be some confusion between the `controller_name_for_metric` method name saying "name" and the variable holding its result saying "path".

Reply:
Updated in 53639679fc7b9da07442ed96b68449c2dbe983a7